source/_integrations/template.markdown

```diff
@@ -330,6 +330,36 @@ sensor:
 
 Useful entities to choose might be `sensor.date` which update once per day or `sensor.time` which updates once per minute.
 
+Sometimes it is reasonable to convert an entity-less template that operates with time into one with entities to get your sensor value updated automatically:
+
+{% raw %}
+
+```yaml
+# Original entity-less template sensor that WON'T be updated automatically
+- platform: template
+    sensors:
+      house_asleep:
+        value_template: >-
+          {{ is_state('binary_sensor.anybody_home', 'off') and (now().hour|float > 22 or now().hour|float < 6)) }}
+```
+
+{% endraw %}
+
+{% raw %}
+
+```yaml
+# Converted template sensor that uses sensor.time entity and WILL BE be updated automatically on every sensor.time update (i.e every minute)
+# DON'T FORGET TO ENABLE sensor.time IN YOUR configuration.yaml!
```

Comment:
We should shout (full capitalization of part of a sentence) and we should not put notes into code comments. 

Code comments make no sense in a documentation page, since well... there is space to just write documentation...

Reply:
got it